challenges/objects-arrays.js

```diff
@@ -89,32 +127,50 @@ The zoo wants to display both the scientific name and the animal name in front o
 
 */
 const animalNames = [];
+const namesofani = zooAnimals.forEach((animalinfo) => {
+  animalNames.push(`Name: ${animalinfo.animal_name}, Scientific: ${animalinfo.scientific_name}.`);
+});
 console.log(animalNames);
 
+
 /* Request 2: .map()    
+const lowerCase = [];
+const tolowercase = zooAnimals.map((lower) => {
+  return `Name: ${lower.animal_name.toUowerCase}`;
+});
+console.log(lowerCase);
 
 The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 
 */
 
 const lowerCase = [];
-console.log(lowerCase); 
+const tolowercase = zooAnimals.map((lower) => {
+  lowerCase.push(`Name: ${lower.animal_name.toLowerCase()}`);
+});
+console.log(lowerCase);
```

Comment:
Yes! It seems you've got a good grasp on array methods and pushing data into new arrays! Way to go!